experiments/utils/download.py

```diff
@@ -21,6 +22,37 @@
 
 """
 
+DEFAULT_ZFILL = 4
+
+
+def _create_file_name(
+        dp: DataPoint,
+        suffix: str = ".txt",
+        zfill: int = DEFAULT_ZFILL
+        ) -> str:
+    """Creates a file name with suffix based on the datapoint
+
+    The file name created is well suited for alphabetical
+    ordering in a filebrowser, and allows for multiple different
+    `DataPoint`s for one ParticipantSession.
+
+    :param dp: The DataPoint for which we want to create a download name
+
+    :param suffix: The suffix use to append to the file name, note that
+                   this doesn't include the '.'.
+
+    :param zfill: The amount of zero padding applied to the subject_id and
+                  `DataPoint.number`. By default, it accomodates for [0001-9999]
+                  alphabetical ordering.
+    """
+    readable_title = re.sub(r'\s+', "-", dp.experiment.title.strip().lower())
```

Comment:
I would be even more strict and replace any `\W` instead of `\s`

Reply:
I had to lookup \W, but that's nice indeed. I'll change it.

Reply:
fixed in https://github.com/UiL-OTS-labs/web-experiment-datastore/pull/39/commits/244099b0d87cc466a3a4d800e348eb94799d2bf8